src/cargo/core/features.rs

```diff
@@ -687,6 +687,26 @@ macro_rules! unstable_cli_options {
                 fields
             }
         }
+
+        #[cfg(test)]
+        mod test {
+            #[test]
+            fn ensure_sorted() {
+                // This will be printed out if the fields are not sorted.
+                let location = std::panic::Location::caller();
+                println!(
```

Comment:
Maybe this?

```suggestion
                crate::drop_println!(
```

Reply:
Should we switch to https://rust-lang.github.io/rust-clippy/master/index.html#/print_stdout and https://rust-lang.github.io/rust-clippy/master/index.html#/print_stderr instead of the `internals` test?

In our clippy config, we could then set `allow-print-in-tests = true`

This way we have better control, get the feedback locally as well as CI, and allow debugging

Reply:
https://github.com/rust-lang/cargo/pull/12246 fixed this. I will merge this soon.